deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -674,14 +688,56 @@ cephObjectStores:
 #    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
 #    # unmapOptions: force
 #
-#    # RBD image format. Defaults to "2".
-#    imageFormat: "2"
+#      # RBD image format. Defaults to "2".
+#      imageFormat: "2"
+#
+#      # RBD image features, equivalent to OR'd bitfield value: 63
+#      # Available for imageFormat: "2". Older releases of CSI RBD
+#      # support only the `layering` feature. The Linux kernel (KRBD) supports the
+#      # full feature complement as of 5.4
+#      # imageFeatures: layering,fast-diff,object-map,deep-flatten,exclusive-lock
+#      imageFeatures: layering
+#    allowVolumeExpansion: true
+#    reclaimPolicy: Delete
+#
+#  - name: rook-ceph-block-2
```

Comment:
I'd suggest let's just keep a single example in the list. How about if we remove this example?

Reply:
yea fixed